README.md

```diff
@@ -24,7 +24,7 @@ This will run jupyter and all search engines in Docker containers. Check that ea
 - Jupyter: [localhost:8888](localhost:8888)
 
 ## You want to build your own LTR notebooks
-git 
```

Comment:
I believe that is acting as a section header in this context.

Reply:
![image](https://github.com/user-attachments/assets/b34f8249-c280-4deb-a18c-259b8e41b4f2)
maybe some other formatting needed then?

Reply:
yeah, that's weird, it doesn't belong